users/miles2go/babblePaste.h

```diff
@@ -0,0 +1,358 @@
+/*  A library to output the right key shortcut in any common app. 
+Given a global variable babble_mode to show the environment and a 
+key that calls the paste macro, do the right type of paste. 
+
+Setting the bable_mode is done by another macro, or TBD interaction with the host. 
+
+Huge thanks to https://en.wikipedia.org/wiki/Table_of_keyboard_shortcuts
+and jeebak & algernon's keymap
+*/
+
+#pragma once
+#include "action_layer.h"
+#include "quantum_keycodes.h"
```

Comment:
If you include `quantum.h`, that should get both of these, actually.  And other important stuff.